gammapy/maps/geom.py

```diff
@@ -122,7 +120,10 @@ def find_and_read_bands(hdu, header=None):
             name = re.search("(.+)_MIN", cols[0]).group(1)
         else:
             name = cols[0]
-
+        intp = "INTERP%i" % (i+1)
```

Comment:
"intp" is a pretty cryptic name. Maybe call it "interp_key" or something a bit more descriptive?

Reply:
The keyword length is limited to 8 characters unfortunately.

Reply:
I just meant the Python variable name, not the FITS header key name.

Reply:
Changed to `interp_key`